Block/Adminhtml/System/Config/CardFormCustomization/Theme.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Omise\Payment\Block\Adminhtml\System\Config\CardFormCustomization;
+
+class Theme
+{
+
+    public function getCustomizationDesign($theme, $customDesign)
```

Comment:
IMO, `getFormDesign` would be more appropriate since it's only form design abd returns default design if  `$customDesign` is empty.